app/controllers/fileUpload/WhatYouWillNeedController.scala

```diff
@@ -54,6 +54,24 @@ class WhatYouWillNeedController @Inject()(
         case _ => None
       }
 
+      val (templateDownloadLink, instructionsDownloadLink): (Json.JsValueWrapper, Json.JsValueWrapper) = (psr, chargeType) match {
+          case (Some(true), ChargeTypeAnnualAllowance) =>
```

Comment:
I have a feeling this code could be shortened - quite a bit of repetition of code here

Reply:
It can probably be replaced with one line:-
val (templateDownloadLink, instructionsDownloadLink) = (controllers.routes.FileDownloadController.templateFile(chargeType, psr).url,
        controllers.routes.FileDownloadController.instructionsFile(chargeType, psr).url)

Reply:
One liner adopted